.github/workflows/check-combinations.yml

```diff
@@ -0,0 +1,47 @@
+name: "Check combinations.json"
+on:
+  schedule:
+  # Run at midnight on the first of every month
+  # https://crontab.guru/once-a-month
+  - cron: "0 0 1 * *"
+
+  push:
```

Comment:
This action runs once a month and on every push. When this is ready to merge, "push" should be removed: we don't want it running that often. Push is turned on for now for testing purposes only.

Also, we should revisit how often this should run. I think that once a month may be too infrequent; what do you think about once a week?